lib/contestant.rb

```diff
@@ -0,0 +1,24 @@
+class Contestant
+  attr_reader :first_name, :last_name, :age, :spending_money, :state_of_residence, :game_interests
+
+  def initialize(info)
+    @first_name = info[:first_name]
+    @last_name = info[:last_name]
```

Comment:
it's no problem to save the first and last name here, but nothing in the interaction pattern suggests you need it, so you could have just captured `full_name` here instead